ui/main/src/app/modules/feed/components/map/map.component.ts

```diff
@@ -26,15 +26,20 @@ import {OpfabMap} from 'app/modules/share/map/opfab-map';
 })
 export class MapComponent extends OpfabMap implements OnInit, OnDestroy, AfterViewChecked {
     private filteredLightCardStore: FilteredLightCardsStore;
+    private initialZoomToLocation: any;
 
     constructor(
         private translateService: TranslateService,
         private superChangeDetector: ChangeDetectorRef,
-        private router: Router
+        private router: Router,
+        private route: ActivatedRoute
     ) {
         super(translateService, superChangeDetector);
         this.targetElementId = 'ol-map';
         this.filteredLightCardStore = OpfabStore.getFilteredLightCardStore();
+        this.route.queryParams.subscribe((params) => {
```

Comment:
Maybe add a takeUntil(this.unsubscribe$)

Not sure if it is necessary here , i prefer to be careful